extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -310,11 +315,33 @@ export class SchemaCompareResult {
 		});
 	}
 
+	private createApplyButton(view: azdata.ModelView) {
+
+		this.applyButton = view.modelBuilder.button().withProperties({
+			label: localize('schemaCompare.updateButton', 'Apply'),
```

Comment:
Update or Apply?

Reply:
Andrew suggested "Apply," but it can always be changed if it's confusing